### ISSUE TITLE
file updates after the first one are not detected

### ISSUE THREAD

**Issue by lx4r** [lx4r/markdownfox#13]
reported on Linux Mint, reproduced on Ubuntu

**Comment by lx4r** [comment]
I did some more testing and I was only able to reproduce the bug using gedit on Ubuntu 17.10. The issue appeared both when I opened the file in gedit first and then in MarkdownFox and the other way round.  
I was not able to reproduce the issue with nano, Visual Studio Code or Atom (all also on Ubuntu 17.10).  
A possible cause for the issue could be that gedit handles file saving different than the other editors and maybe removes the old file and saves a new one. I imagine such behaviour could cause problems with the file's inode and Node's `fs.watch` (see [docs](https://nodejs.org/docs/latest/api/fs.html#fs_inodes)). Unfortunately I have no idea how to fix this except using the more inefficient `fs.watchFile` and I don't want to do that.

TL;DR: Don't know how to fix this issue that seems to be specific for gedit. Help appreciated.
